ingeniamotion/configuration.py

```diff
@@ -1127,9 +1134,15 @@ def get_vendor_id(self, servo: str = DEFAULT_SERVO, axis: int = DEFAULT_AXIS) ->
 
         Raises:
             TypeError: If the read vendor ID has the wrong type.
+            ValueError: If axis ID not in [0, 1].
 
         """
-        register = self.VENDOR_ID_COCO_REGISTER if axis == 0 else self.VENDOR_ID_REGISTER
+        if axis == 0:
+            register = self.VENDOR_ID_COCO_REGISTER
+        elif axis == 1:
+            register = self.VENDOR_ID_REGISTER
+        else:
+            raise ValueError(f"Vendor ID cannot be retrieved for {axis=}")
```

Comment:
Thanks for improving unknown cases, this should be our default standards!

Reply:
In fact, that was probably done because of the multiaxis drives. Probably dictionary subnodes should be checked.